<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QMainWindow" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="configpic">
   <property name="styleSheet">
    <string notr="true">#configpic{border-image:url(:/images/configwindow/setting_background.png)}</string>
   </property>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnReturn{border-image:url(:/images/return.png)}
QPushButton::pressed#btnReturn{border-image:url(:/images/return-.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMusic">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnMusic{border-image:url(:/images/configwindow/setting_music_on.png)}
QPushButton::pressed#btnMusic{border-image:url(:/images/configwindow/setting_music_on-.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSound">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnSound{border-image:url(:/images/configwindow/setting_sound_on.png)}
QPushButton::pressed#btnSound{border-image:url(:/images/configwindow/setting_sound_on-.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="musicSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	border: 0px solid white;
	height:40px;
}
/*已滑过的槽部分*/
QSlider::sub-page:horizontal {
	background-color:rgb(100, 33, 0) ;
border-radius: 20px;
}

/*未滑过的槽部分*/
QSlider::add-page:horizontal {
	background-color: rgb(255, 188, 155);
border-radius: 20px;
}
/*滑块*/
QSlider::handle:horizontal 
{
	
	background-color:rgb(170, 170, 255);
	width: 32px;
	margin: -14px  0px -14px 0px; 
border-radius: 10px;
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="soundSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	border: 0px solid white;
	height:40px;
}
/*已滑过的槽部分*/
QSlider::sub-page:horizontal {
	background-color:rgb(100, 33, 0) ;
border-radius: 20px;
}

/*未滑过的槽部分*/
QSlider::add-page:horizontal {
	background-color: rgb(255, 188, 155);
border-radius: 20px;
}
/*滑块*/
QSlider::handle:horizontal 
{
	
	background-color:rgb(170, 170, 255);
	width: 32px;
	margin: -14px  0px -14px 0px; 
border-radius: 10px;
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBgd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>背景</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDia">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>宝石</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnBod">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>棋盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnMsc">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>音效</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRecharge">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>600</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>钞能力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/configwindow/board.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widgetDim" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="BtnDim1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 爱的魔法</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnDim2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 动物乐园</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnDim3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 敬请期待</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBgd" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="BtnBgd1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 默认</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnBgd2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 星空</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnBgd3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 魔法</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBod" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="BtnBod1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 棋盘1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnBod2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 棋盘2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnBod3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 棋盘3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetMsc" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="BtnMsc1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 音乐1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnMsc2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 音乐2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BtnMsc3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
	font-size:17px;
	color:Red;
	font-family:幼圆;
	text-align: center;
}</string>
     </property>
     <property name="text">
      <string> 音乐3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetQR" native="true">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>170</y>
      <width>350</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
  background-color: #f0f0f0;
}</string>
    </property>
    <widget class="QLabel" name="QRpic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>350</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resource/res.qrc">:/images/configwindow/QRcode.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
  background-color: #000000;
	color: white;
}</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <zorder>btnReturn</zorder>
   <zorder>btnMusic</zorder>
   <zorder>btnSound</zorder>
   <zorder>musicSlider</zorder>
   <zorder>soundSlider</zorder>
   <zorder>btnBgd</zorder>
   <zorder>BtnDia</zorder>
   <zorder>BtnBod</zorder>
   <zorder>BtnMsc</zorder>
   <zorder>BtnRecharge</zorder>
   <zorder>label</zorder>
   <zorder>widgetBgd</zorder>
   <zorder>widgetBod</zorder>
   <zorder>widgetDim</zorder>
   <zorder>widgetMsc</zorder>
   <zorder>widgetQR</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
